profiles/use.desc

```diff
@@ -72,6 +73,7 @@ dvb - Add support for DVB (Digital Video Broadcasting)
 dvd - Add support for DVDs
 dvdr - Add support for DVD writer hardware (e.g. in xcdroast)
 eds - Enable support for Evolution-Data-Server (EDS)
+egl - Enable EGL support
```

Comment:
This one doesn't add any explanation beyond what one can already guess from the flag's name.

Reply:
Added an expansion and short explanation based on Wikipedia.